### DISCUSSION TITLE
Triggers flow again i new Tab

### DISCUSSION THREAD

**Discussion by Sotis1** [getPraxis/FELauncher#9] [Q&A]
Hi,

Thanks for the component!

But I have one issue, I have implement it in a salesforce console solution and if I fire a row action from the enhanced grid table it works fine but if I then change tab to another record which has this component also on the record page it fires the flow again for the old record.

What should I do to fix this?

/Jens

**Comment by ohadios** [comment]
Jens
We are aware of this issue (it's a known issue on the Issues tab: https://github.com/getPraxis/FELauncher/issues/3)
We hope to get this fixed in the next few weeks.

Ohad

**Reply by Sotis1** [reply]
Sorry! should have checked the Issues first /Jens
